airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/schemas/transactions.json

```diff
@@ -208,10 +246,11 @@
                       "type": ["null", "object"],
                       "properties": {
                         "currency_code": {
```

Comment:
Please update the currency code and value properties according to the [documentation](https://developer.paypal.com/docs/api/transaction-search/v1/#definition-money).